Update README.md (#1)
blindpirate/gradle-legacy-osgi-plugin: README.md

## A replacement for deprecated Gradle 5 osgi plugin

> `OSGI` plugin has been removed from Gradle core plugins since Gradle 6, yet some builds still depend on it.

Reviewer: ```suggestion
`osgi` plugin has been removed from Gradle core plugins since Gradle 6, yet some builds still depend on it.
```

> This repo extracts the legacy `OSGI` plugin (and unit test) from Gradle 5.6.4 source code.

Reviewer: ```suggestion
This repo extracts the legacy `osgi` plugin (and unit test) from Gradle 5.6.4 source code.
```

> If you're still using the deprecated `OSGI` Gradle plugin ('osgi'), you need to migrate to using ('com.github.blindpirate.osgi').

Reviewer: ```suggestion
If you're still using the deprecated `osgi` Gradle plugin in Gradle 6, you can use this plugin (`com.github.blindpirate.osgi`) as an alternative.
```
